Intersect.Client.Framework/Gwen/Control/Base.cs

```diff
@@ -1648,6 +1648,14 @@ public virtual void MoveTo(int x, int y, bool aligning = false)
             SetBounds(x, y, Width, Height);
         }
 
+        /// <summary>
+        ///     Sets the control position based on ImagePanel
+        /// </summary>
+        public virtual void SetPosition(ImagePanel _icon)
```

Comment:
If you want to define a position here, let it be the Base position and not the ImagePanel
It is not good for us to bring those who inherit the base into their father
And if we can't define coordinates when using a "SetPosition" then maybe the name of the function should be something else, something more linked to what the function does, otherwise the name "SetPosition" itself loses meaning, we cannot choose where

Reply:
changed to Base, i wouldn't change the virtual void name though, the summary explains what it does and the propose is the same: to set the control's position, but based on the icon position